### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by bohanjason**
1. require Python 3.8+
2. database user permissions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 ## Setup
 
-To install python dependencies:
+To build the Agent, you need `Python 3.8+`. Install python dependencies:
```
Comment (NealKaviratna) [inline README.md:21]:
```suggestion
To run the Agent, you will need an enviorment with `Python 3.8+`.

Install python dependencies:
```

Comment (NealKaviratna) [reply README.md:21]: nit python isn't built, and these two things can be separate lines I think.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,9 @@ python3 -m driver.main --aws-region <region> --db-identifier <db-identifier-with
 
 If you are using postgres you will need to set the `POSTGRES_OTTERTUNE_DB_NAME` env variable if you are using a postgres database.
 
+## Permissions
+To collect metrics from the database, you need to [configure database settings](https://docs.ottertune.com/info/connect-your-database-to-ottertune/add-database#configure-database-settings), and [grant permissions](https://docs.ottertune.com/info/connect-your-database-to-ottertune/add-database/guarded-public-access#create-the-database-user-and-grant-permissions-1) to the database user.
+
```
Comment (NealKaviratna) [inline README.md:46]: should we add a `running from source` section instead of putting this in the `Development` section?

Comment (bohanjason) [reply README.md:48]: I think we also need such permissions during the development. Do we want to merge this first and add a new `Running from source` section later ?

Comment (NealKaviratna) [reply README.md:48]: sounds good to me

**Review by NealKaviratna — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -18,7 +18,9 @@
 
 ## Setup
 
-To install python dependencies:
+To run the Agent, you will need an enviorment with `Python 3.8+`.
```
Comment (NealKaviratna) [inline README.md:21]:
```suggestion
To run the Agent, you will need an environment with `Python 3.8+`.
```

oops sorry I added a typo

**Pull request merged**